eth/stagedsync/default_stages.go

```diff
@@ -326,6 +339,7 @@ var DefaultForwardOrder = UnwindOrder{
 	stages.LogIndex,
 	stages.TxLookup,
 	stages.Finish,
+	stages.Completed,
```

Comment:
please don't rename last stage. because much software already expect that stages.Finish will always be the last one.

Reply:
If I do that it'll break reverse comparability... if a user puts: `STOP_BEFORE_STAGE=Finish` it'll stop before being finished. If i move that logic to the step before (like first draft of this pr did), the user will need to put `STOP_BEFORE_STAGE=Finalize` to get same results.

The first draft of this PR had a bunch of test failures, so I suspect it's because some tests depend on `Finish` updating new RPC blocks.

Reply:
no. stage.Finish was introduced to guarantee that it always be the last one. many places use stage.Finish progress as an "erigon overall progress" marker. for example: `eth_Syncing` (`cmd/rpcdaemon/commands/eth_system.go:APIImpl.Syncing`) or Grafana dashboard. 

Compatibility of STOP_BEFORE_STAGE flag is not important - it's very rare used undocumented thing.